lib/kb_uploadmethods/Utils/UploaderUtil.py

```diff
@@ -251,6 +271,29 @@ def _upload_file_path(self, params):
 
       return result
 
+  def _load_gff_fasta_file_path(self, params):
+      """
+      _load_gff_fasta_file_path: load gff and fasta to workspace as genome using files from user's staging area
+      
+      fasta_file: fasta file from user's staging area
+      gff_file: gff file from user's staging area
+      genome_name: output genome object name
+      workspace_name: workspace name that genome will be stored to
+      """
+      log ('---> running UploaderUtil._load_gff_fasta_file_path')
+      
+      workspace_name_or_id = params.get('workspace_name')
+      
+      #for now must use workspace name, but no ws_id_to_name() function available
+      if str(workspace_name_or_id).isdigit():
+          raise ValueError('"' + workspace_name_or_id + '" parameter is a workspace id and workspace name is required')
```

Comment:
not unittest, in your actual methods. 

Then in unittest, test to see if bad params triggers the error.